### ISSUE TITLE
PL version problem

### ISSUE THREAD

**Issue by Zielon** [ThibaultGROUEIX/NeuralJacobianFields#2]
Hello, it seems that the repository does not specify the PL version. Installing the current version > 2.0.0 is not working. Even if I install an older one I get ```The provided lr scheduler `MultiStepLR` doesn't follow PyTorch's LRScheduler API. You should override the LightningModule.lr_scheduler_step hook with your own logic if you are using a custom LR scheduler.``` error. Which makes it very problematic to use. Could you provide the PL version or fix the issuse with scheduler? Thank you!

**Comment by ThibaultGROUEIX** [comment]
Hi ! Thanks for the first issue !
Please see : https://github.com/ThibaultGROUEIX/NeuralJacobianFields/blob/main/py39_njf_requirements.txt

Thanks,
Thibault

**Comment by Duotun** [comment]
If use PL version > 2.0.0, the problem could be fixed through changing import logics to lightning_fabrics:
[source code changes](https://lightning.ai/docs/pytorch/1.9.0/_modules/pytorch_lightning/utilities/seed.html)
